deploy/scripts/configure.sh

```diff
@@ -118,11 +125,72 @@ az postgres flexible-server \
 az aks get-credentials --resource-group ${AKS_RESOURCE_GROUP} --name ${AKS_CLUSTER_NAME} --context ${AKS_CLUSTER_NAME} --overwrite-existing
 kubectl config set-context ${AKS_CLUSTER_NAME}
 
-echo "creating $AKS_NAMESPACE namespace"
-envsubst < "${PRJ_ROOT}/deploy/kube_yaml/namespace.yaml" | kubectl apply -f -
-
-echo "deploying stacfastapi"
-envsubst < ${PRJ_ROOT}/src/stac_fastapi_k8s/app-stacfastapi-deployment.tpl.yaml | kubectl -n $AKS_NAMESPACE apply -f -
-
-echo "deploying aks-ingest"
-envsubst < ${PRJ_ROOT}/src/stac_ingestion/aks-ingest-deployment.yaml | kubectl -n $AKS_NAMESPACE apply -f -
\ No newline at end of file
+NS=$(kubectl get namespace $AKS_NAMESPACE --ignore-not-found);
+if [[ "$NS" ]]; then
+    echo "Skipping creation of ${AKS_NAMESPACE} namespace in k8s cluster as it already exists"
+else
+    echo "Creating ${AKS_NAMESPACE} namespace in k8s cluster"
+    kubectl create namespace ${AKS_NAMESPACE}
+fi;
+
+echo "Install KEDA"
+kubectl apply -f https://github.com/kedacore/keda/releases/download/v2.8.0/keda-2.8.0.yaml
```

Comment:
KEDA 2.9 was just released: https://keda.sh/blog/2022-12-12-keda-2.9.0-release/

Reply:
I will take up this change after merging the current PR. Capturing this change to the Sprint board in the meantime.